readme.md

```diff
@@ -106,6 +106,13 @@ Default: `new Map()`
 
 Use a different cache storage. Must implement the following methods: `.has(key)`, `.get(key)`, `.set(key, value)`, and optionally `.clear()`. You could for example use a `WeakMap` instead or [`quick-lru`](https://github.com/sindresorhus/quick-lru) for a LRU cache.
 
+##### cacheRejection
+
+Type: `boolean`<br>
+Default: `false`
+
+Set to `true` if you want to cache rejected promises.
```

Comment:
I think `Set to true if you want to` is moot. The default is defined, so we can just describe what it does:

> Cache rejected promises.